akka-docs/rst/java/remoting.rst

```diff
@@ -599,3 +599,11 @@ special configuration that sets both the logical and the bind pairs for remoting
       }
    }
   }
+
+Keep in mind that local.address must be in one of private network ranges: 
```

Comment:
There is no requirement that the bind-hostname address needs to be in the private network range. It certainly will be the case in most of the setups, but some networks could use some other address space for the bind-hostname ([even EC2 VPC docs](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Subnets.html#vpc-sizing-ipv4) recommend but do not mandate this range). The only requirement is that there needs to be an address translation from the `hostname` to `bind-hostname` in the network. I think we can add this note with the change from *must* to *will almost always be*. WDYT?